src/redux/services/hrms/employeeJoining/sourceMaster/index.js

```diff
@@ -0,0 +1,64 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { toast } from 'react-toastify';
+import customAxios from '../../../../../http/axios';
+
+export const getSourceMasterListThunk = createAsyncThunk('sourceMaster/getSourceList', async () => {
+  try {
+    const response = await customAxios.get(`/sourceMaster`);
+    if (response?.status === 200 || response?.status === 201) {
+      if (response?.data?.status === 1) {
+        return { data: response?.data?.data, msg: response?.data?.message };
+      } else {
+        toast.error(response?.data?.message);
+      }
+    }
+  } catch (error) {
+    toast.error(error?.response?.data?.message);
+    return Promise.reject(error?.response?.data?.message);
+  }
+});
+export const addSourceMasterThunk = createAsyncThunk(
+  'sourceMaster/addSource',
+  async ({ formData, onSuccessHandler, onErrorHandler }) => {
+    try {
+      const response = await customAxios.post(`/sourceMaster`, formData);
+      if (response?.status === 200 || response?.status === 201) {
+        if (response?.data?.status === 1) {
+          onSuccessHandler();
+          toast.success(response?.data?.message);
```

Comment:
We have common success color for toaster all over the project.
keep it common.

Reply:
The common success color for the toaster has been implemented throughout the entire project to maintain consistency. This change will be included in the next PR.